### PR TITLE
Created a hello microverse repository

### PR DESCRIPTION
**Pull request by mnaibei**
Review of my repo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+
```
Comment (IngDavidTM) [inline .gitignore:105]: - [ ] Good job adding `node_modules` to your `.gitignore` file :+1:, but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should make sure that also test.md is actually ignored by git and is not present in your GitHub repository :100:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @mnaibei  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] While you did a wonderful job creating your pull request :100:, I think it would be better if the title and description are more descriptive telling you what changes are inside, because it would make your repository looks more professional :+1: 

- [ ] Good job adding the README file to your project :+1:, but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should add descriptive README file to your project using [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project :100: 

- [ ] As a general requirement you should create a `LICENSE` file because it would make your project open source so that others are free to use, change, and distribute the software. You can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)

- [ ] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mnaibei** [comment]
Created a hello microverse project with HTML&CSS
Added Linters

**Comment by mnaibei** [comment]
Hello @IngDavidTM I have changed my PR title but I am unable to change its description. Is there a way to do it or would I have to make a whole new PR?

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @mnaibei 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**